Themis/Views/Exercises/ExerciseView.swift

```diff
@@ -90,7 +91,6 @@ struct ExerciseView: View {
         Button {
             Task {
                 await assessmentVM.initRandomSubmission(exerciseId: exercise.id)
-                UndoManagerSingleton.shared.undoManager.removeAllActions()
```

Comment:
Why did you remove this? It seems unrelated to errors.

Reply:
Yes I decided to remove this here as it somehow got left there, it should've been removed before when refactoring the undo/redo.